src/openms/source/FORMAT/HANDLERS/MzIdentMLDOMHandler.cpp

```diff
@@ -1188,7 +1189,12 @@ namespace OpenMS
                 // check for retention time or scan time entry
                 if (cvit->first == "MS:1000894" || cvit->first == "MS:1000016") //TODO use subordinate terms which define units
                 {
-                  pep_id_->back().setRT(cvit->second.front().getValue().toString().toDouble()); // TODO convert if unit is minutes
+                  double rt = boost::lexical_cast<double>(cvit->second.front().getValue());
```

Comment:
better to use the toDouble() here I think because the lexical cast is quite slow